packages/plugin-relay/src/utils/connections.ts

```diff
@@ -231,7 +231,7 @@ function parseCurserArgs(options: ResolveOffsetConnectionOptions) {
 }
 
 // biome-ignore lint/suspicious/noRedeclare: <explanation>
-type NodeType<T> = T extends (infer N)[] | Promise<(infer N)[] | null> | null ? N : never;
+export type NodeType<T> = T extends (infer N)[] | Promise<(infer N)[] | null> | null ? N : never;
```

Comment:
This would need a much less generic name to be exported, and I'm not sure if it actually makes sense to export as is.